client/src/components/SearchResultListItemImage/SearchResultListItemImage.js

```diff
@@ -0,0 +1,9 @@
+const { default: styled } = require("@emotion/styled");
```

Comment:
Why don't you use "import" instead of require like in the other files?

Reply:
Wow! Thanks for pointing this out! It is weird, I thought I dit the auto-import and something went wrong! I will fix it!